notice/notice.go

```diff
@@ -0,0 +1,17 @@
+package notice
+
+type Notice struct {
+	Type    NoticeType
+	Message string
+	Details string
+}
+
+// NoticeType represents the level of severity of a Notice.
+type NoticeType int
```

Comment:
Since we're already in qualified in the package `notice` just call this `Type` -- it will be used as `notice.Type` everywhere and `notice.NoticeType` stutters.